packages/angular_devkit/build_angular/src/dev-server/schema.json

```diff
@@ -77,6 +77,54 @@
       "type": "boolean",
       "description": "Show a warning when deploy-url/base-href use unsupported serve path values.",
       "default": true
+    },
+    "optimization": {
+      "type": "boolean",
+      "description": "Defines the optimization level of the build.",
+      "default": false
```

Comment:
These options cannot have a default, otherwise the default will always override the browser target values.

Reply:
Done.